lib/constants/enums.ts

```diff
@@ -10,4 +10,9 @@ enums.orderingDirections = {
   ASC: 'ASC',
 };
 
+enums.statuses = {
```

Comment:
The enum name need to be better associated with `BaseClient`. maybe it should be called `clientConnectionStatuses`, or simply `connectionStatuses`. Merely `statuses` won't work since it's a global enum now.